command_factory.py

```diff
@@ -0,0 +1,25 @@
+from functions import *
+
+
+class CommandFactory:
+    def __init__(self, args):
+        self.command = args.command
+        self.schema = args.schema
+        self.database = args.database
+        self.table = args.table
+        self.primary_key = args.primary_key
+        self.parameter = args.parameter
+        self.value = args.value
+
+    def create(self):
+        cmd = self.command
+        if cmd == "create_dir":
```

Comment:
as we agreed, why creat_dir still exist?

Reply:
we still working on this